### PR TITLE
Update host-based-security-workflows.rst (IDFGH-11090)

### PR DESCRIPTION
**Pull request by walerii**
Fixed typos in the code examples which caused troubles when trying to follow the secure boot workflow and improved the grammar.

### CODE REVIEW

**Comment by AdityaHPatwardhan** [comment]
@walerii Thanks for the PR, I will take it up internally. 
I have some small changes requested, I would fix them up if required.

File: docs/en/security/host-based-security-workflows.rst [hunk 1]
```diff
@@ -8,17 +8,17 @@ Host Based Security Workflows
 Introduction
 ------------
 
-It is recommended to have an uninterrupted power supply while enabling security features on ESP32 SoCs. Power failures during secure manufacturing process could cause issues that are hard to debug and, in some cases, may cause permanent boot-up failures.
+It is recommended to have an uninterrupted power supply while enabling security features on ESP32 SoCs. Power failures during the process of secure boot enabling could cause issues that are hard to debug and, in some cases, may cause permanent boot-up failures.
```
Comment (AdityaHPatwardhan) [inline docs/en/security/host-based-security-workflows.rst:11]: the document is generic and related to manufacturing processes. Secure boot is a part of that so I think the previous line should be fine. Other than that the MR looks really good to me, Thanks for fixing the documentation

**Review by Harshal5 — APPROVED** [review]
LGTM!

File: docs/en/security/host-based-security-workflows.rst (hunk 1 above)
Comment (walerii) [reply docs/en/security/host-based-security-workflows.rst]: Thanks for your feedback. I have updated the document according to your comment.

**Comment by AdityaHPatwardhan** [comment]
@walerii Can you just merge both commits into one.

**Comment by mahavirj** [comment]
@walerii Can you please squash the commits?

**Comment by walerii** [comment]
@mahavirj unfortunately I am not sure how to do this.
Additional guidance is highly appreciated.

**Comment by AdityaHPatwardhan** [comment]
@walerii Can you please mention that software interface you have used to raise the PR?
Are you using Git CLI or some software like Git Desktop?
Based on that we can give you some guidelines.

**Comment by AdityaHPatwardhan** [comment]
sha=b47536e955906a7c1f60dd725ae86c4b9c14e089

**Comment by AdityaHPatwardhan** [comment]
sha=b5589db377ebb2b0489593b82477bca6a53fa384

**Pull request closed without merging**